### PR TITLE
Add radon (and mando)

### PR DESCRIPTION
**Pull request by igortg**
Add *radon* lib and its dependency, *mando*

### CODE REVIEW

File: recipes/xenon/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
```
Comment (isuruf) [inline recipes/xenon/meta.yaml]: This package has entrypoints. You can add them using conda-build, https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#python-entry-points

Comment (isuruf) [inline recipes/xenon/meta.yaml]: This package has entrypoints. You can add them using conda-build, https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#python-entry-points

**Comment by igortg** [comment]
@isuruf entry points added for radon and xenon. Please, take a look.

File: recipes/xenon/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  imports:
+    - xenon
+  commands:
+    - xenon -v
+
+about:
+  home: https://github.com/rubik/xenon
+  license: MIT
+  license_family: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```
Comment (isuruf) [inline recipes/xenon/meta.yaml]: Did you open an issue upstream about including the License in source distributions?

Comment (igortg) [reply recipes/xenon/meta.yaml]: Done:
https://github.com/rubik/xenon/issues/20

**Pull request merged**